adapters/pulsepoint.go

```diff
@@ -0,0 +1,171 @@
+package adapters
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/prebid/openrtb"
+	"github.com/prebid/prebid-server/pbs"
+	"golang.org/x/net/context/ctxhttp"
+)
+
+type PulsePointAdapter struct {
+	http         *HTTPAdapter
+	URI          string
+	usersyncInfo *pbs.UsersyncInfo
+}
+
+// adapter name
+func (a *PulsePointAdapter) Name() string {
+	return "PulsePoint"
+}
+
+// used for cookies and such
+func (a *PulsePointAdapter) FamilyName() string {
+	return "pulsepoint"
+}
+
+func (a *PulsePointAdapter) GetUsersyncInfo() *pbs.UsersyncInfo {
+	return a.usersyncInfo
+}
+
+// parameters for pulsepoint adapter.
+type PulsepointParams struct {
+	PublisherId int    `json:"cp"`
+	TagId       int    `json:"ct"`
+	AdSize      string `json:"cf"`
+}
+
+func (a *PulsePointAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
+	ppReq := makeOpenRTBGeneric(req, bidder, a.FamilyName())
+	for i, unit := range bidder.AdUnits {
```

Comment:
Confirming that pulsepoint handles multiple adunits per call - please make sure a unit test hits this as it often causes issues